modules/rubiconAnalyticsAdapter.js

```diff
@@ -608,7 +612,7 @@ let rubiconAdapter = Object.assign({}, baseAdapter, {
             delete bid.error; // it's possible for this to be set by a previous timeout
             break;
           case NO_BID:
-            bid.status = args.status === BID_REJECTED ? 'rejected-ipf' : 'no-bid';
+            bid.status = args.status === BID_REJECTED ? BID_REJECTED_IPF : 'no-bid';
```

Comment:
I was thinking `args.status === BID_REJECTED_IPF` as the constant.  Cause what if new code starts rejecting bids for a reason other than ipf?   But this would require updates in other places.

Reply:
but we can fix this later.  for now this is ok